### PR TITLE
Create revue-francaise-d-administration-publique.csl

### PR DESCRIPTION
**Pull request by rpierronnet**
CSL Style for french journal "revue française d'administration publique"

### CODE REVIEW

File: revue-francaise-d-administration-publique.csl [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-last">
+    <names variable="author">
+      <name name-as-sort-order="all" sort-separator=", " delimiter=" ; " delimiter-precedes-last="always" font-variant="normal" font-weight="normal"/>
+      <label form="short" prefix=" " text-case="lowercase"/>
+      <substitute>
+        <names variable="editor">
+          <name name-as-sort-order="all" sort-separator=", " delimiter=", " delimiter-precedes-last="always" initialize-with="." font-variant="small-caps"/>
```
Comment (adam3smith) [inline revue-francaise-d-administration-publique.csl]: is this on purpose? Editors and authors are formatted differently (e.g. small caps vs. not, semicolong vs. comma, etc.)? That seems very odd.

File: revue-francaise-d-administration-publique.csl [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+        </names>
+      </substitute>
+    </names>
+  </macro>
+  <!-- Same as author-last except that name is not initialized to sort on first name too -->
+  <macro name="author-sort">
+    <names variable="author">
+      <name name-as-sort-order="all" sort-separator=", " delimiter=" " delimiter-precedes-last="always" font-variant="small-caps"/>
+      <label form="short" prefix=" " text-case="lowercase"/>
+      <substitute>
```
Comment (adam3smith) [inline revue-francaise-d-administration-publique.csl]: similarly here

File: revue-francaise-d-administration-publique.csl [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+          <text term="edition" form="long"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="citation-locator">
+    <group>
```
Comment (adam3smith) [inline revue-francaise-d-administration-publique.csl]: no need for this group

**Comment by rmzelle** [comment]
Thanks! Sorry for not merging this sooner.

**Pull request merged**